### ISSUE TITLE
[C-630] Add a description to attributes

### ISSUE THREAD

**Issue by joanreyero** [CrowdDotDev/crowd.dev#506]
The `isHireable` attribute coming from GitHub has already confused people a couple of times. We could add a new field to attributes, *description*, and show it as a Tooltip if it exists

---

Created via [Raycast](https://www.raycast.com)

<sub>From [SyncLinear.com](https://synclinear.com) | [C-630](https://linear.app/crowddotdev/issue/C-630/add-a-description-to-attributes)</sub>

**Comment by peoray** [comment]
cc @yeganathan18, @joanagmaia, can you help here? @joanreyero seems to be busy ;)

**Comment by joanagmaia** [comment]
Hey @peoray, this is a specific attribute we get from Github.

For more context:
- `isHireable` is added to the `attributes` json of a member. Each attribute as a `default` value and a value per platform (if it exists). So for instance. `isHireable` would be like this:
```
isHireable:  {github: false, default: false}
```
And in the UI it would be displayed like this (we render the default value)
![Screenshot 2023-09-11 at 18 34 27](https://github.com/CrowdDotDev/crowd.dev/assets/20134207/c82aa802-d0da-42a2-b78f-910816ab7499)

This issue asks that the backend adds a `description` field to the attribute. The frontend would then display it if it existed. Visually, it could be an information icon next to the attribute name, and when hovered, it would show a tooltip with that description.

This ticket would require backend and frontend work

**Comment by a0m0rajab** [comment]
Adding extra context here: this ticket does not have a dummy data that could show it from the first time we open the project. (at least that was my case)

**Comment by yeganathan18** [comment]
@a0m0rajab Just to make sure I've got this straight, you're indicating that there's no dummy data included in the project for testing purposes, is that right?

**Comment by peoray** [comment]
yeah  @yeganathan18 
do you have suggestions on how to achieve that?

**Comment by a0m0rajab** [comment]
Yes, that's right, to test this we need to have an account already connected with GitHub and has users from it. 

It would much valuable if we can use it with dummy data .

**Comment by yeganathan18** [comment]
Integrating with GitHub helps with the data, but I'm unsure about the feasibility without the envs. Would you mind creating the GitHub envs by yourselves? 

If I were in your shoes, I'd manually add the GitHub envs by setting them up through my account and establishing a connection with GitHub. In that way, we could get some data for fixing the issue 🙂

**Comment by a0m0rajab** [comment]
You are right but I don't have an organisation in GitHub with few people to test, this is the issue here. Otherwise I would have done that

Sent from mobile device


________________________________

**Comment by yeganathan18** [comment]
@a0m0rajab Sorry for that, I'll check with the team and see if there is a way we could really add some dummy data for testing purposes.
